src/tests/containerizer/cgroups2_tests.cpp

```diff
@@ -210,6 +221,119 @@ TEST_F(Cgroups2Test, ROOT_CGROUPS2_EnableAndDisable)
   EXPECT_EQ(0u, enabled->count("cpu"));
 }
 
+// Combination of a path and access flags.
+typedef pair<string, int> Access;
+
+using DeviceControllerTestParams = tuple<
+    vector<devices::Entry>, 
+    vector<devices::Entry>,
+    vector<Access>, 
+    vector<Access>>;
```

Comment:
hm.. it's a little weird to define another Access here when there's already a devices::Entry::Access we could be using, maybe something like this instead?

```
// Combination of a path and access flags for open()ing the device.
typedef pair<string, int> OpenTest;

using DeviceControllerTestParams = tuple<
    vector<devices::Entry>, 
    vector<devices::Entry>,
    vector<OpenTest>,  // Opens that should be allowed.
    vector<OpenTest>>; // Opens that should be denied.
```

Reply:
Sure, yeah :+1:

Reply:
Renamed it `OpenArgs`, because it's the arguments for `os::open`.